test/Microsoft.TestPlatform.TestUtilities/Class1.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Microsoft.VisualStudio.TestPlatform.Common.ExtensionFramework;
+using Microsoft.VisualStudio.TestPlatform.Common.SettingsProvider;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Logging;
+using Microsoft.VisualStudio.TestPlatform.Utilities.Helpers.Interfaces;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+namespace Microsoft.TestPlatform.TestUtilities
+{
+    #region Testable implementation
+
+    public class TestableTestPluginCache : TestPluginCache
```

Comment:
nit: rename file to match the class name.

Reply:
done